src/Umbraco.Web.UI/Umbraco/config/lang/cy.xml

```diff
@@ -0,0 +1,2766 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
The path of these language files are incorrect (even that we talk about changing the casing of the UmbracoFolder to this:
![image](https://user-images.githubusercontent.com/1561480/132474929-a9d1d0ea-99ac-4915-b8b9-eff5015990b5.png)

Reply:
That's probably the reason for it showing up as new file 😓